imap_processing/swapi/swapi_cdf_attrs.py

```diff
@@ -0,0 +1,106 @@
+"""Shared attribute values for SWAPI CDF files."""
+
+from imap_processing.cdf.defaults import GlobalConstants
+from imap_processing.cdf.global_attrs import (
+    AttrBase,
+    GlobalDataLevelAttrs,
+    GlobalInstrumentAttrs,
+    ScienceAttrs,
+)
+from imap_processing.swapi import __version__
+
+text = (
+    "The Solar Wind and Pickup Ion (SWAPI) instrument "
+    "measures several different elements of the solar "
+    "wind, including hydrogen (H) and helium (He) ions,"
+    " and, on occasion, heavy ions produced by large "
+    "events from the Sun. SWAPI also measures "
+    "interstellar pickup ions (PUIs), whose material "
+    "comes from beyond our solar system and moves along"
+    " with the solar wind. Its data will provide "
+    "information about the local ion conditions, such "
+    "as temperature, density, and speed. This will also"
+    " be used in the I-ALiRT data stream allowing space"
+    " weather to be measured in real-time. The data from"
+    " SWAPI will be valuable for understanding how the "
+    "solar wind changes in response to the Sun's behavior"
+    " over time. SWAPI's first-ever high time resolution "
+    "measurements of helium PUIs will provide new insights"
+    " into physical processes that accelerate charged "
+    "particles and shape and change our global heliosphere."
+    "See https://imap.princeton.edu/instruments/swapi for "
+    "more details."
+)
+
+swapi_base = GlobalInstrumentAttrs(
+    __version__,
+    "SWAPI>The Solar Wind and Pickup Ion",
+    text,
+    "Particles (space)",
+)
+
+# dimensions attributes
+energy_dim_attrs = AttrBase(
+    validmin=0,
+    validmax=72,
+    format="I2",
+    var_type="support_data",
+    fieldname="Energy Step",
+    catdesc="Energy step number.",
+    label_axis="Energy Step",
+)
+
+# science data attributes
+counts_attrs = ScienceAttrs(
+    validmin=GlobalConstants.INT_FILLVAL,
+    validmax=GlobalConstants.INT_MAXVAL,
+    format="I12",
+    units="Counts",
+    label_axis="Counts",
+    display_type="spectrogram",
+    catdesc=("Primary, secondary CEM or coincidence counts."),
+    fieldname="Counts",
+    fill_val=GlobalConstants.INT_FILLVAL,
+    var_type="data",
+    depend_0="epoch",
+    depend_1="energy",
+)
+
+# Uncertainty data is in Float and requires a different fill value,
+# format, min value.
+uncertainty_attrs = ScienceAttrs(
+    validmin=GlobalConstants.DOUBLE_FILLVAL,
+    validmax=GlobalConstants.INT_MAXVAL,
+    format="E13.5",
+    label_axis="Counts Uncertainty",
+    display_type="spectrogram",
+    catdesc=("Uncertainty in the counts."),
+    fieldname="Counts Uncertainty",
+    fill_val=GlobalConstants.DOUBLE_FILLVAL,
+    var_type="data",
+    depend_0="epoch",
+    depend_1="energy",
+)
+
+compression_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=1,
+    format="I1",
+    label_axis="Compression Flag",
```

Comment:
Probably just "Flag" for the label_axis here.

Reply:
Is "Flag" too general?

Reply:
I believe the idea is that, if you were plotting it via CDAWeb, the FIELDNAM would appear above/below the plot as the title. In this case, the plot title would be "Compression Flag".  

And then LABLAXIS just labels the y axis on that plot.  So because the FIELDNAM is on the plot, the viewer would know what "Flag" is referring to here.

Reply:
I see. That makes sense.